src/components/LandOperatorPage/LandOperatorForm/LandOperatorForm.tsx

```diff
@@ -72,18 +73,34 @@ export default class LandOperatorForm extends React.PureComponent<Props, State>
 
     return (
       <Form className="LandOperatorForm">
-        <Field
-          placeholder="0x..."
-          label={t('operator_page.address')}
-          className={classes.join(' ')}
-          value={address}
-          onChange={this.handleChange}
-          loading={loading}
-          action={isRevokable ? (revoked ? t('operator_page.undo') : t('operator_page.revoke')) : undefined}
-          onAction={isRevokable ? (revoked ? this.handleUndo : this.handleRevoke) : undefined}
-          error={hasError}
-          message={hasError ? t('operator_page.invalid_address') : undefined}
-        />
+        {isEnsAddressEnabled ? (
+          <AddressField
+            placeholder="0x..."
+            label={t('operator_page.address')}
+            className={classes.join(' ')}
+            value={address}
+            onChange={this.handleChange}
+            loading={loading}
+            action={isRevokable ? (revoked ? t('operator_page.undo') : t('operator_page.revoke')) : undefined}
+            onAction={isRevokable ? (revoked ? this.handleUndo : this.handleRevoke) : undefined}
+            error={hasError}
```

Comment:
Will `hasError` work with the new field?

Reply:
yess it will take the value coming for props or the value calculated on the field